### PR TITLE
Package structure

### PR DESCRIPTION
**Pull request by timowest**
Related to https://github.com/querydsl/querydsl/issues/1039

### CODE REVIEW

**Comment by Shredder121** [comment]
The parent of `querydsl-docs` doesn't exist.
The version specified is 3.2.1, shouldn't we equalize that?

**Comment by timowest** [comment]
@Shredder121 Could you take a look? This is close to ready. 
The introduction of querydsl-sql-spatial is the only bigger change.
The package names are now IMHO more consistent as they match now the artifactId.

**Comment by timowest** [comment]
@Shredder121 The maven plugin annotations should be fixed now.

**Comment by Shredder121** [comment]
There are a few renames that missed your attention, but what is special about `/querydsl-apt/etc/META-INF/services/javax.annotation.processing.Processor`?

File: querydsl-apt/etc/META-INF/services/javax.annotation.processing.Processor [hunk 1]
```diff
@@ -1 +1 @@
-com.mysema.query.apt.EntityAnnotationProcessor
\ No newline at end of file
+querydsl.apt.EntityAnnotationProcessor
```
Comment (Shredder121) [inline querydsl-apt/etc/META-INF/services/javax.annotation.processing.Processor]: @timowest This file in particular.

**Comment by timowest** [comment]
Which other mismatches did you find?

**Comment by Shredder121** [comment]
In the documentation I found a link that doesn't work anymore, I am fixing it right now.
Moreover, there are too many occurrences of query changed to querydsl. Which I am also fixing right now.

**Comment by timowest** [comment]
I changed now some of the occurences.

**Comment by Shredder121** [comment]
I reviewed the entire diff, and made changes accordingly.
I personally finished my sweep, so I am finished with the review.

How does it look on your end?

**Comment by timowest** [comment]
I will review it in the evening. Is it ok for you if I combine some commits to get a cleaner PR?

**Comment by Shredder121** [comment]
In essence yes, but out of curiosity which commits?

**Comment by timowest** [comment]
Maybe everything except the first, because the remaining are mostly fixes.

**Comment by Shredder121** [comment]
That's okay.

**Comment by timowest** [comment]
It took some time to get it right, but it's IMHO quite good. The remaining commits are three distinct changes to the codebase.

**Comment by timowest** [comment]
It took some time to get it right, but it's IMHO now quite good. The remaining commits are three distinct changes to the codebase.

**Comment by Shredder121** [comment]
I agree, it is indeed now quite good.

**Comment by timowest** [comment]
@Shredder121 Shall we merge then?

**Comment by Shredder121** [comment]
Well, the build succeeded.
Speaking of build, could you also setup the build in cloudbees for QUERYDSL_3?

**Comment by Shredder121** [comment]
Haha, you beat me to it.
Yes, we can merge now.

**Comment by timowest** [comment]
Yes, I will add another build to Cloudbees.

**Pull request merged**